ui-service/src/main/frontend/hotel-booking-ui/src/pages/HomePage.tsx

```diff
@@ -1,11 +1,15 @@
 import HomePageExampleComponent from "../components/HomePageExampleComponent";
+import SearchHotelComponent from "../components/search-components/SearchComponent.tsx";
 
 const HomePage: React.FC = () => {
-  return (
-    <HomePageExampleComponent>
-      <h1>"Here is a placeholder for homepage components"</h1>
-    </HomePageExampleComponent>
-  );
+    return (
+        <>
+            <HomePageExampleComponent>
```

Comment:
The example component can be removed now since the search section is now the actual first component